src/packages.ml

```diff
@@ -30,6 +30,39 @@ let mlds_by_package_def =
          ))
        |> Package.Name.Map.of_list_reduce ~f:List.rev_append)
 
+let packages_by_dir_def =
```

Comment:
it seems to me that we don't need this code anymore

Reply:
Removed.